### ISSUE TITLE
IQ8 series Envoy

### ISSUE THREAD

**Issue by Pheerink** [H4nsie/EnphaseEnvoy#7]
I replaced all microconveters to IQ8H (previous IQ7+) and the pulgin gives me error messages. Is this due to the other protocol ? This is what I get:

2023-05-02 14:13:13.629 Error: Envoy panels: Call to function 'onHeartbeat' failed, exception details:
2023-05-02 14:13:13.630 Error: Envoy panels: Traceback (most recent call last):
2023-05-02 14:13:13.630 Error: Envoy panels: File "/home/pi/domoticz/plugins/EnphaseEnvoy/plugin.py", line 316, in onHeartbeat
2023-05-02 14:13:13.630 Error: Envoy panels: _plugin.onHeartbeat()
2023-05-02 14:13:13.630 Error: Envoy panels: File "/home/pi/domoticz/plugins/EnphaseEnvoy/plugin.py", line 275, in onHeartbeat
2023-05-02 14:13:13.630 Error: Envoy panels: self.getData()
2023-05-02 14:13:13.630 Error: Envoy panels: File "/home/pi/domoticz/plugins/EnphaseEnvoy/plugin.py", line 124, in getData
2023-05-02 14:13:13.630 Error: Envoy panels: Domoticz.Debug("Got sessionId '" + response_data['session_id'] + "' from Enphase Enlighten.")

Any idee ?

**Comment by 0crap** [comment]
I would try the native Domoticz Enphase implementation, to see if it behaves any better.

**Comment by H4nsie** [comment]
Error at line 124 is due to a failed connection to: https://enlighten.enphaseenergy.com/login/login.json I think it I a down server at Enphase.

It should not have anything to do with other inverters. When a new serialnumber of an inverter is found, it will automatically create a device in Domoticz. (I installed 3 extra panels in juni and they appeared in Domoticz automatically).

Since my Envoy was updated to V7 and is supported by the Domoticz native implementation, I am not maintaining this source anymore.
